packages/mgt-components/src/components/mgt-taxonomy-picker/mgt-taxonomy-picker.ts

```diff
@@ -0,0 +1,401 @@
+/**
+ * -------------------------------------------------------------------------------------------
+ * Copyright (c) Microsoft Corporation.  All Rights Reserved.  Licensed under the MIT License.
+ * See License in the project root for license information.
+ * -------------------------------------------------------------------------------------------
+ */
+
+import { TermStore } from '@microsoft/microsoft-graph-types';
+import { html, TemplateResult } from 'lit';
+import { property, state } from 'lit/decorators.js';
+import { MgtTemplatedComponent, mgtHtml, customElement } from '@microsoft/mgt-element';
+import { strings } from './strings';
+import { fluentCombobox, fluentOption } from '@fluentui/web-components';
+import { registerFluentComponents } from '../../utils/FluentComponents';
+import '../../styles/style-helper';
+
+registerFluentComponents(fluentCombobox, fluentOption);
+
+/**
+ * Web component that allows a single entity pick from a generic endpoint from Graph. Uses mgt-get.
+ *
+ * @fires {CustomEvent<TermStore.Term>} selectionChanged - Fired when an option is clicked/selected
+ * @export
+ * @class MgtTaxonomyPicker
+ * @extends {MgtTemplatedComponent}
+ */
+// @customElement('mgt-picker')
+@customElement('taxonomy-picker')
+export class MgtTaxonomyPicker extends MgtTemplatedComponent {
+  protected get strings() {
+    return strings;
+  }
+
+  /**
+   * The termSetId of the term set whose children to get.
+   *
+   * @type {string}
+   * @memberof MgtTaxonomyPicker
+   */
+  @property({
+    attribute: 'termset-id',
```

Comment:
```suggestion
    attribute: 'term-set-id',
```

This will make the react script to convert it to `termSetId`.